sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ExpressionConverter.java

```diff
@@ -517,13 +517,7 @@ public RexNode convertResolvedLiteral(ResolvedLiteral resolvedLiteral) {
         ret = convertValueToRexNode(resolvedLiteral.getType(), resolvedLiteral.getValue());
         break;
       default:
-        throw new RuntimeException(
-            MessageFormat.format(
-                "Unsupported ResolvedLiteral type: {0}, kind: {1}, value: {2}, class: {3}",
-                resolvedLiteral.getType().typeName(),
-                kind,
-                resolvedLiteral.getValue(),
-                resolvedLiteral.getClass()));
+        throw new RuntimeException("Unsupported ResolvedLiteral type.");
```

Comment:
This change makes error message not clear as before, right?

Reply:
It is certainly less verbose, but I feel almost every error message in this file could be simplified to `Unsupported type` without affecting the clarity of the message to our end users.

Reply:
It's very useful for development actually. And for users, at least what kind of ResolvedLiteral should be thrown to remind where could be wrong in user queries.

Reply:
I would prefer a mixed approach:
1. give user the unsupported kind.
2. print log info of more information to help development.